samples/cpp_synchronization/src/main.cpp

```diff
@@ -133,7 +133,7 @@ void coop_thread_entry(void)
 	}
 }
 
-void main(void)
+int main(void)
 {
 	struct k_timer timer;
 
```

Comment:
This is allowed by the compiler, the return value is some undefined value, but the kernel ignores it anyway.